start-docker-nix-build-slave

```diff
@@ -62,17 +62,20 @@ CONF
 
 if ! test -f "$HOME/.ssh/config" || ! grep "$docker_machine_name" "$HOME/.ssh/config" > /dev/null; then
   cecho ">>> Adding an entry to $HOME/.ssh/config for $docker_machine_name"
-  echo "$SSHCONF" > "$HOME/.ssh/config"
+  mkdir -p "$HOME/.ssh"
+  echo "$SSHCONF" >> "$HOME/.ssh/config"
 else
   cecho ">>> User SSH already contains entry for $docker_machine_name in $HOME/.ssh/config"
 fi
 
 if [ -S /nix/var/nix/daemon-socket/socket ]; then
-  if ! sudo test -f "/root/.ssh/config" || ! sudo grep "$docker_machine_name" "/root/.ssh/config" > /dev/null; then
-    cecho ">>> Adding an entry to /root/.ssh/config for $docker_machine_name"
-    echo "$SSHCONF" | sudo tee "/root/.ssh/config" > /dev/null
+  [[ "$OSTYPE" = "darwin"* ]] && ROOT_HOME=/var/root || ROOT_HOME=/root
```

Comment:
```suggestion
  ROOT_HOME=~root
```

Reply:
wow, I didn't realize that existed actually